zstacklib/zstacklib/utils/ovn.py

```
@@ -299,3 +306,214 @@ def addUplink(self, portPciMap, bondMode, lacpmode, ip, netmask, brName="br-phy"
+            if os.path.exists(VSWITCHD_PID_PATH):
+                with open(VSWITCHD_PID_PATH, 'r') as f:
+                    pids[1] = int(f.read().strip())
+        except OSError as err:
+            logger.error("OSError: {}".format(err))
+        finally:
+            return pids[0] != -1 and pids[1] != -1
+
+    @bash.in_bash
+    def getOvsOtherConfig(self, key):
+        try:
+            ret = bash.bash_r(CtlBin + "get Open_vSwitch . other_config:{}".format(key)).strip("\n").strip('"')
```

Pylint (3.3.7)

[error] Instance of 'int' has no 'strip' member

(E1101)